docs/tutorial/workers.md

```diff
@@ -0,0 +1,166 @@
+
+## Setting up the worker and work pool
+
+For this tutorial you will create a **Docker** type work pool via the CLI. 
+
+Using the **Docker** work pool type means that all work sent to this work pool will run within a dedicated docker container using a docker client available to the worker.
+
+!!! tip "Other work pool types"
+    There are work pool types for all major managed code execution platforms, such as Kubernetes services or serverless computing environments such as AWS ECS, Azure Container Instances, and GCP Cloud Run.
+    
+    These are expanded upon in the [Guides](/guides) section.
+
+### Create the work pool
+
+In your terminal run the following command to set up a **Docker** type work pool. 
+<div class="terminal">
+```bash
+prefect work-pool create --type docker my-docker-pool
+```
+</div>
+Let’s confirm that the work pool was successfully created by running the following command in the same terminal. You should see your new `my-docker-pool` in the output list.
+<div class="terminal">
+```bash
+prefect work-pool ls 
+```
+</div>
+Finally, let’s double check that you can see this work pool in your Prefect UI. Navigate to the Work Pool tab and verify that you see `my-docker-pool` listed.
+
+When you click into the `my-docker-pool`, select the "Work Queues" tab. You should see a red status icon listed for the default work queue signifying that this queue is not ready to submit work. Using and configuring work queues is an advanced deployment mode. You can learn more about them in the [work queue documentation.](/concepts/work-pools/#work-queues) 
+
+To get the work queue healthy and ready to submit flow runs, you need to start a worker.
+
+### Starting a worker
+
+Workers are a lightweight polling process that kick off scheduled flow runs on a certain type of infrastructure (like Docker). To start a worker on your laptop, open a new terminal and confirm that your virtual environment has `prefect` installed.
+
+Run the following command in this new terminal to start the worker:
+<div class="terminal">
+```bash
+prefect worker start --pool my-docker-pool
+```
+</div>
+You should see the worker start - it's now polling the Prefect API to request any scheduled flow runs it should pick up and then submit for execution. You’ll see your new worker listed in the UI under the worker tab of the Work Pool page with a recent last polled date. You should also be able to see a `Healthy` status indicator in the default work queue under the work queue tab - progress!
+
+You will need to keep this terminal session active in order for the worker continue to pick up jobs. Since you are running this worker locally, the worker will terminate if you close the terminal. Therefore, in a production setting this worker should be running as a daemonized or managed process. See next steps for more information on this.
```

Comment:
```suggestion
You will need to keep this terminal session active in order for the worker to continue to pick up jobs. Since you are running this worker locally, the worker will terminate if you close the terminal. Therefore, in a production setting this worker should be running as a daemonized or managed process. See next steps for more information on this.
```